### PR TITLE
Feature new subgait format

### PR DESCRIPTION
**Pull request by Olavhaasie**
Closes PM-500

## Description
This PR implements reading and writing the new subgait format. See project-march/gait-files#188 for more info on the new format. Previously we used the `Subgait.msg` description as subgait format. Since I have now changed the subgait format this message was no longer necessary and removed it from the gait scheduler action topic. Most of the changes are from the subgait files used in the tests so you can filter to those out :wink:

## Changes
* Remove `Subgait.msg` from the gait scheduler action
* Add reading and writing new format

### CODE REVIEW

File: march_shared_classes/src/march_shared_classes/gait/subgait.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Subgait(object):
     joint_class = JointTrajectory
 
     def __init__(self, joints, duration, gait_type='walk_like', gait_name='Walk', subgait_name='right_open',
-                 version='First try', description='Just a simple gait'):
+                 version='First try', description='Just a simple gait', *args):
```
Comment (Roelemans) [inline march_shared_classes/src/march_shared_classes/gait/subgait.py:18]:
```suggestion
                 version='First try', description='Just a simple gait', *args):
```
I believe this *args is not necessary (also wasn't in there before). Also remove the one in the call in line 113

**Comment by Olavhaasie** [comment]
> Most of the testing subgait file still have the references on the time_from_start variables (for example in the march sharded classes and march gait selection test subgaits) :sad:

I don't know what you mean, could you give an example? `time_from_start` is still a valid variable in the new format. See project-march/gait-files#188 for the complete format

**Comment by JorisWeeda** [comment]
> > Most of the testing subgait file still have the references on the time_from_start variables (for example in the march sharded classes and march gait selection test subgaits) :sad:
> 
> I don't know what you mean, could you give an example? `time_from_start` is still a valid variable in the new format. See [project-march/gait-files#188](https://github.com/project-march/gait-files/pull/188) for the complete format

Take a look at the subgait format in the testing gait files inside the gait (for example `march_ws/src/march/march_gait_selection/test/testing_gait_files/stairs_up/left_swing`). Same issue as describe in [roel his comment](https://github.com/project-march/gait-files/pull/188#issuecomment-656184130).

**Comment by Olavhaasie** [comment]
@JorisWeeda They have been removed

**Comment by JorisWeeda** [comment]
> @JorisWeeda They have been removed

:fire:

**Pull request merged**